src/config/selectors.ts

```diff
@@ -13,6 +13,7 @@ export const cancelDialogButtonCypress = 'cancelDialogButtonCypress';
 export const confirmDialogButtonCypress = 'confirmDialogButtonCypress';
 export const messagesContainerCypress = 'messagesContainerCypress';
 export const inputTextFieldTextAreaCypress = 'inputTextFieldTextArea';
+export const charCounterCypress = 'charCounterCypress';
```

Comment:
I don't know which convention you're following, I'm personally not adding `Cypress` to the string, as it's obviously set to `data-cy`, but it's a minor comment.